core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -197,15 +195,15 @@ export class ContributionOpportunitiesBackendApiService {
   }
 
   async fetchReviewableTranslationOpportunitiesAsync(
-      topicName: string,
+      topicName?: string,
```

Comment:
Can we make this so either '' or ALL is used as the sentinel, but not both? I'm also fine with filing a follow-up issue for it, but we really should standardize this.

Reply:
To clarify, the caller should pass in `topicName = 'All'` rather than making `topicName` an optional parameter. Is this correct?

Reply:
Yes that's correct, and let's make 'ALL' a named constant rather than a raw string in the code. I.e. callers should pass in TOPIC_SENTINEL_NAME_ALL.

Reply:
Done.